packages/frontend/web-ui/src/app/store/features/authSlice.js

```diff
@@ -2,27 +2,34 @@ import { createSlice } from '@reduxjs/toolkit';
 import { createAuthByCredentials } from '../thunk/createAuthByCredentials';
 import { createAuthByToken } from '../thunk/createAuthByToken';
 
+const STATUS_AUTH_AUTHENTICATED = 'authenticated';
+const STATUS_AUTH_NOT_AUTHENTICATED = 'not-authenticated';
+const STATUS_AUTH_CHECKING = 'checking';
+
 function createAuthPendingReducer(state) {
-  state.status = 'checking';
+  state.status = STATUS_AUTH_CHECKING;
   state.token = null;
   state.errorMessage = null;
 }
 
 function createAuthFulfilledReducer(state, action) {
-  state.status = 'not-authenticated';
+  state.status = STATUS_AUTH_NOT_AUTHENTICATED;
 
-  if (action.payload.statusCode === 200) {
-    state.status = 'authenticated';
-    state.token = action.payload.body.jwt;
-  } else if (action.payload.statusCode === 422) {
-    state.errorMessage = 'Unprocessable operation. Try again.';
-  } else {
-    state.errorMessage = 'Ups... something strange happened. Try again?';
+  switch (action.payload.statusCode) {
+    case 200:
+      state.status = STATUS_AUTH_AUTHENTICATED;
+      state.token = action.payload.body.jwt;
+      break;
+    case 422:
```

Comment:
Docs are wrong, there's no 422 response. Instead, there's a 401 response when credentials are wrong.

Reply:
Do I make the change now or I can wait to make later?

Reply:
Done!